app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+        self, name: str, health: int = 100, hidden: bool = False
```

Comment:
Quando existir muitos parâmetros (normalmente mais de três) ou quando os nomes dos parâmetros forem muito grandes, tente quebrar um parâmetro por linha. Isso ajuda na legibilidade do código.

```suggestion
        self,
        name: str,
        health: int = 100,
        hidden: bool = False
```